RecoEgamma/PhotonIdentification/src/PhotonMIPHaloTagger.cc

```diff
@@ -290,17 +290,11 @@ std::vector<double> PhotonMIPHaloTagger::GetMipTrailFit(const reco::Photon* phot
            
 
         //Lets Initialize them first for each iteration 
-        res    = 0.0;
-        res_sq = 0.0;
-        wt     = 0.0; 
```

Comment:
Sincerely I thing we are going well beyond what to expect form the average developers and what we SHALL expect from the compiler's optimizer:
Assigning zero to a variable does not harm (and in many other contest we ask developers to initialize variables)
And the compiler is free to optimize it away if its dependency analysis discover that it is redundant...